app/vmctl/vm/vm.go

```diff
@@ -281,6 +286,19 @@ func (im *Importer) flush(ctx context.Context, b []*TimeSeries) error {
 	return nil
 }
 
+func (im *Importer) getHTTPClient() *http.Client {
+	var client *http.Client
+	if im.insecureSkipVerify {
+		tr := &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		}
+		client = &http.Client{Transport: tr}
+	} else {
+		client = http.DefaultClient
+	}
+	return client
+}
```

Comment:
I think we can create a client when we init Importer, and we can make `http.Client` as a part of the Importer struct. It will create only one client.

In your realization, it will create a client on each request.

@hagen1778 wdt?

Reply:
Agree. I think it is better to make http.client a member of Importer struct. The client needs to be created in NewImporter function call and used for all interactions afterwards instead of http.DefaultClient.